src/app/scripts/cac/control/cac-control-directions-list.js

```diff
@@ -118,7 +137,7 @@ CAC.Control.DirectionsList = (function ($, Handlebars, UserPreferences, Utils) {
 
     function toggle() {
         if ($container.hasClass('hidden')) {
-            show()
+            show();
```

Comment:
No problem. Related: want to take a look at load_events.py? I just ran the python linter and saw some warnings there.